client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/ContactDateCard/InteractionCard/AddressGrid/AddressGrid.tsx

```diff
@@ -18,32 +15,12 @@ const AddressGrid : React.FC<Props> = (props: Props) : JSX.Element => {
                 <Grid item xs={6}>
                     <FormInput fieldName='כתובת'>
                         <Typography variant='caption'>
-                            {interaction.locationAddress.description}
+                            {interaction.locationAddress ? (interaction.locationAddress as any).description : 'לא הוזן מיקום'}
```

Comment:
Maybe you can create a type for this?

Reply:
types are really disorganised right now since locationAddress type is not up to date with the db.
it should be googlegeocodeapi | null ant not as it is currently - an object with and address property and entrance and floor property.
the type for locationAddress will be refactored soon